Makefile

```diff
@@ -15,7 +15,7 @@ UPDATE_FILE_DIR = build/update_files
 LEAF_DATA = ../leaf-data/current
 
 # Arguments passed to mksys whenever it is invoked.
-MKSYS_ARGS_COMMON = --object-dir=build/$@ --output-dir=$(UPDATE_FILE_DIR)
+MKSYS_ARGS_COMMON = -v -C -O2 --object-dir=build/$@ --output-dir=$(UPDATE_FILE_DIR)
```

Comment:
Do we really want `-v`?

Reply:
nope.  That was an accident.  Thanks for catching that